app/services/workflows/non_passported_workflow.rb

```diff
@@ -145,16 +145,15 @@ def collate_and_assess_gross_income(assessment:, self_employments:, partner_self
       # TODO: make the Collators::DisposableIncomeCollator increment/sum to existing values so order of "collation" becomes unimportant
       def partner_disposable_income_assessment(assessment:, gross_income_subtotals:, applicant_person_data:, partner_person_data:)
         applicant = PersonWrapper.new is_single: false,
-                                      submission_date: assessment.submission_date,
-                                      person_data: applicant_person_data,
-                                      employments: assessment.employments,
-                                      gross_income_summary: assessment.applicant_gross_income_summary
+                                      person_data: applicant_person_data
```

Comment:
think it's still to much to pass the whole person_data in here when all we want is the dependants?